modules/ROOT/examples/swift/code-snippets/SampleCodeTest.swift

```diff
@@ -142,7 +142,32 @@ class SampleCodeTest {
         }
         // end::document-listener[]
     }
-
+    
+    func dontTestDocumentExpiration() throws {
+        database = self.db
+        
+        // tag::document-expiration[]
+        // Purge the document one day from now
+        let ttl = Calendar.current.date(byAdding: .day, value: 1, to: Date())
+        try database.setDocumentExpiration(withID: "doc123", expiration: ttl)
+        
+        // Reset expiration
+        try db.setDocumentExpiration(withID: "doc1", expiration: nil)
+        
+        // Query documents that will be expired in less than five minutes
+        let oneDayFromNow = Date(timeIntervalSinceNow: 60 * 5).timeIntervalSince1970
```

Comment:
probably you mean, `fiveMintsFromNow`?

Reply:
```suggestion
        let fiveMinutesFromNow = Date(timeIntervalSinceNow: 60 * 5).timeIntervalSince1970
```

Reply:
@jayahariv Good catch, I've fixed that for all languages.